ci/Jenkinsfiles/test-trigger.groovy

```diff
@@ -42,7 +42,7 @@ pipeline {
 
   parameters {
     booleanParam(name: 'IS_PROMOTION', defaultValue: false, description: 'Check if mimicking a promotion instead of a build')
-    string(name: 'BUILD_DESCRIPTION', defaultValue: '', description: 'The description to mimick.\nSample for build: \'Build 11.3.49\'.\nSample for promotion: \'Release 11.3 from build 11.3.49\'.')
+    string(name: 'BUILD_DESCRIPTION', defaultValue: '', description: 'The description to mimick.\nSample for build: \'Build 11.3.49\'.\nSample for promotion: \'Release 11.3 from build 11.3.49\' or \'Release 11.3.49\'.')
```

Comment:
NIT: versions like `11.3.49` are for Nx TP ? If yes, since the 11 branch has been given up, another example with 2021.x.y would be more talkative..

Reply:
Agreed, there are multiple references to 11.x, but since the principle is the same I did not bother updating them I admit

Reply:
Done with #152